sampler/README.md

```diff
@@ -471,7 +473,7 @@ $$
 p(\mathbf{x}; \mathbf{p}) = \frac{\Gamma((\sum_{i=1}^k x_i)+1)}{\Gamma(x_1+1)\cdots{}\Gamma(x_k+1)}p_1^{x_1}\cdots{}p_k^{x_k},
 $$
 
-#### Working in "log-space"
+#### Working in "Log-Space"
```

Comment:
Hmm, you're right, though I haven't seen any alternate terms for this concept. Maybe "working with log values" would be clearer and avoid confusion with the complexity class?